commit 96df67164e11ad67a1745f9ee70d404331b73d51 (refs/changes/05/5305/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-01-03T11:05:45-08:00 (9 months ago)
    
    TRITON-760 imgapi-external-manta-setup and imgapi-manta-setup were broken by sdc-clients 11.x update in TRITON-53 (missed some SAPI client usage)
    Reviewed by: Angela Fong <angela.fong@joyent.com>
    Approved by: Angela Fong <angela.fong@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index c54e93c..1e89260 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,10 @@
 # IMGAPI changelog
 
+## 4.6.1
+
+- TRITON-760 Fix imgapi-manta-setup (again, missed some cases) tooling
+  broken in v4.4.0 by TRITON-53.
+
 ## 4.6.0
 
 - TRITON-774 imgapi should allow admin-only file-add from URLs, adds a new
diff --git a/bin/imgapi-manta-setup b/bin/imgapi-manta-setup
index c89f8db..0166483 100755
--- a/bin/imgapi-manta-setup
+++ b/bin/imgapi-manta-setup
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -311,6 +311,7 @@ function generateKey(keyName, callback) {
 function getSdcApp(config, callback) {
     var sapi = new sdcClients.SAPI({
         url: 'http://' + config.sapi_service,
+        version: '~2',
         log: log,
         agent: false
     });
@@ -345,6 +346,7 @@ function ensureImgapiSapiServiceSetup(args, callback) {
         args.sdcApp.metadata.datacenter_name);
     var sapi = new sdcClients.SAPI({
         url: 'http://' + args.config.sapi_service,
+        version: '~2',
         log: log,
         agent: false
     });
diff --git a/package.json b/package.json
index 97dd340..4af7cf3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for Triton Data Center",
-  "version": "4.6.0",
+  "version": "4.6.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
